docs/toc.json

```diff
@@ -178,11 +178,13 @@
 			},
 			{ "docs/how-to-guides/accessibility.md": [] },
 			{ "docs/how-to-guides/internationalization.md": [] },
-			{ "docs/how-to-guides/widgets/README.md": [
-				{ "docs/how-to-guides/widgets/overview.md": [] },
-				{ "docs/how-to-guides/widgets/opting-out.md": [] },
-				{ "docs/how-to-guides/widgets/legacy-widget-block.md": [] }
-			] }
+			{
+				"docs/how-to-guides/widgets/README.md": [
+					{ "docs/how-to-guides/widgets/overview.md": [] },
+					{ "docs/how-to-guides/widgets/opting-out.md": [] },
+					{ "docs/how-to-guides/widgets/legacy-widget-block.md": [] }
+				]
+			}
```

Comment:
This change was a result of simply saving the file in Visual Studio Code. Based on the rest of the file, I think this was simply badly formatted and it makes sense to correct its format here.

Reply:
I tried formatting the file locally and I had the same result 👍 .